subsystems/provisioning/grpc.go

```diff
@@ -16,7 +16,7 @@ func (w *Provisioning) startGRPC() error {
 	bind := PortalBindAddr + ":4772"
 	lis, err := net.Listen("tcp", bind)
 	if err != nil {
-		return errw.Wrapf(err, "error listening on: %s", bind)
+		return errw.Wrapf(err, "listening on: %s", bind)
```

Comment:
A bunch of these scattered around. Errors no need to say the word "error" as it's implicit in the fact that it IS an error. Saw one in the new code, and cleaned up all my own uses of this elsewhere to match.